app/views/homepage/_location_bar.haml

```diff
@@ -1,7 +1,3 @@
--content_for(:location_search) do
-  - if(feature_enabled?(:location_search) && search_engine == :zappy)
-    = javascript_include_tag "https://maps.google.com/maps/api/js?sensor=true&libraries=places"
```

Comment:
This is good :+1: Definitely we shouldn't require the map component ad hoc in partials :+1: